src/huggingface_hub/hf_api.py

```diff
@@ -498,7 +505,7 @@ def __init__(self, **kwargs):
         self.blob_id = kwargs.pop("oid")
         lfs = kwargs.pop("lfs", None)
         if lfs is not None:
-            lfs = BlobLfsInfo(size=lfs["size"], sha256=lfs["oid"], pointer_size=lfs["pointerSize"])
+            lfs = BlobLfsInfo(lfs)
```

Comment:
Related to the comment above, this modification is not needed anymore.

```suggestion
            lfs = BlobLfsInfo(size=lfs["size"], sha256=lfs["oid"], pointer_size=lfs["pointerSize"])
```